src/lib/evas/include/evas_private.h

```diff
@@ -1577,8 +1552,8 @@ _gfx_to_evas_render_op(Efl_Gfx_Render_Op rop)
 }
 #endif
 
-#undef EAPI
-#define EAPI
+#undef EVAS_API
```

Comment:
I think this _undef define_ should be removed to the ensure the same definition for `EVAS_API`  to be used in the files that is using this one as headers.